book/src/dev/release-process.md

```diff
@@ -0,0 +1,117 @@
+
+Older Zebra versions that only support previous network upgrades will never be supported, because they are operating on an unsupported Zcash chain fork.
+
+<a id="updating"></a>
+
+### Supported update paths
+
+You can update to any version of Zebra, provided that the following criteria are met:
+
+* The version you want to update *to* is supported.
+* The version you want to update *from* is within one major version of the version you want to
+    upgrade to.
+
+See [Keeping Up-to-Date](guide/updating "Updating your projects") for more information about updating your Zebra projects to the most recent version.
+
+<a id="previews"></a>
+
+### Preview releases
+
+We let you preview what's coming by providing Release Candidate \(`rc`\) pre-releases for some major releases:
+
+| Pre-release type  | Details |
+|:---               |:---     |
+| Beta              | The release that is under active development and testing. The beta release is indicated by a release tag appended with the `-beta` identifier, such as  `8.1.0-beta.0`.      |
+| Release candidate | A release that is feature complete and in final testing. A release candidate is indicated by a release tag appended with the `-rc` identifier, such as version `8.1.0-rc.0`. |
+
+<a id="frequency"></a>
+
+## Release frequency
+
+We work toward a regular schedule of releases, so that you can plan and coordinate your updates with the continuing evolution of Zebra.
+
+<div class="alert is-helpful">
+
+Dates are offered as general guidance and are subject to change.
+
+</div>
+
+In general, expect the following release cycle:
+
+* A major release for each network upgrade, whenever there are breaking changes to Zebra (by API, severe bugs or other kind of upgrades)
+* Minor releases for significant new Zebra features or severe bug fixes
+* A patch release every few weeks
+
+This cadence of releases gives eager developers access to new features as soon as they are fully developed and pass through our code review and integration testing processes, while maintaining the stability and reliability of the platform for production users that prefer to receive features after they have been validated by Zcash and other developers that use the pre-release builds.
+
+<a id="deprecation"></a>
+
+## Deprecation practices
+
+Sometimes "breaking changes", such as the removal of support for RPCs, APIs, and features, are necessary to:
+
+* add new Zebra features,
+* improve Zebra performance or reliability,
+* stay current with changing dependencies, or
+* implement changes in the \(blockchain\) itself.
+
+To make these transitions as straightforward as possible, we make these commitments to you:
+
+* We work hard to minimize the number of breaking changes and to provide migration tools, when possible
+* We follow the deprecation policy described here, so you have time to update your applications to the latest Zebra binaries, RPCs and APIs
```

Comment:
This is great!

Sometimes we need to turn things off as soon as possible, because they are causing very serious problems. So let's give ourselves an exception here.

```suggestion
* We follow the deprecation policy described here, so you have time to update your applications to the latest Zebra binaries, RPCs and APIs
* If a feature has critical security or reliability issues, and we need to remove it as soon as possible, we will explain why at the top of the release notes
```